nmdc.py

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/python
```

Comment:
The problem with this line is that I use homebrew on OSX. So, my python is at /usr/local/bin/python2.